Waste.API/Services/WasteService.cs

```diff
@@ -28,68 +27,76 @@ public async Task<int> CreateJourney()
                 CreatedBy = "DEVELOPER"
             };
 
-            _wasteContext.WasteJourney.Add(journeyRecord);
-            await _wasteContext.SaveChangesAsync();
+            await _wasteRepository.Add(journeyRecord);
+
             return journeyRecord.Id;
```

Comment:
Do we need to return this if an id is being returned in the sealed Repository class?

Reply:
The id is only generated once it has been saved to the database: it's an auto incrementing field, and therefore we only know about it after we have succesfully saved to the database.